src/Components/Header/Header.js

```diff
@@ -1,7 +1,76 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import MenuDropDown from './MenuDropDown';
+import ProfileModal from './ProfileModal';
+import SearchForm from '../../Pages/Main/SearchForm';
+
+import './Header.scss';
+import ModalPortal from '../../Portal';
 
 const Header = () => {
-  return <div />;
+  const url = document.location.href;
+
+  const [user, setUser] = useState({});
+
+  const [menuOpened, setMenuOpened] = useState(false);
+
+  const [profileClicked, setProfileClicked] = useState(false);
```

Comment:
```suggestion
  const [user, setUser] = useState({});
  const [menuOpened, setMenuOpened] = useState(false);
  const [profileClicked, setProfileClicked] = useState(false);
```
useState를 최상위에서 여러개를 사용할 때 위와 같이 붙여서 관리하는 것이 일종의 컨벤션입니다~